src/automation/azext_automation/aaz/latest/automation/hrwg/hrw/_move.py

```diff
@@ -1,158 +1,163 @@
-                **self.url_parameters
-            )
-
-        @property
-        def method(self):
-            return "POST"
-
-        @property
-        def error_format(self):
-            return "ODataV4Format"
-
-        @property
-        def url_parameters(self):
-            parameters = {
-                **self.serialize_url_param(
-                    "automationAccountName", self.ctx.args.automation_account_name,
-                    required=True,
-                ),
-                **self.serialize_url_param(
-                    "hybridRunbookWorkerGroupName", self.ctx.args.hybrid_runbook_worker_group_name,
-                    required=True,
-                ),
-                **self.serialize_url_param(
-                    "hybridRunbookWorkerId", self.ctx.args.hybrid_runbook_worker_id,
-                    required=True,
-                ),
-                **self.serialize_url_param(
-                    "resourceGroupName", self.ctx.args.resource_group,
-                    required=True,
-                ),
-                **self.serialize_url_param(
-                    "subscriptionId", self.ctx.subscription_id,
-                    required=True,
-                ),
-            }
-            return parameters
-
-        @property
-        def query_parameters(self):
-            parameters = {
-                **self.serialize_query_param(
-                    "api-version", "2021-06-22",
-                    required=True,
-                ),
-            }
-            return parameters
-
-        @property
-        def header_parameters(self):
-            parameters = {
-                **self.serialize_header_param(
-                    "Content-Type", "application/json",
-                ),
-            }
-            return parameters
-
-        @property
-        def content(self):
-            _content_value, _builder = self.new_content_builder(
-                self.ctx.args,
-                typ=AAZObjectType,
-                typ_kwargs={"flags": {"required": True, "client_flatten": True}}
-            )
-            _builder.set_prop("hybridRunbookWorkerGroupName", AAZStrType, ".target_hybrid_runbook_worker_group_name")
-
-            return self.serialize_content(_content_value)
-
-        def on_200(self, session):
-            pass
-
-
-__all__ = ["Move"]
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "automation hrwg hrw move",

```

Comment:
`hrwg hrw`
`hrwg` and `hrw` seem confused

Reply:
HRWG - Hybrid Runbook Worker Group
HRW - Hybrid Runbook Worker.
Our customers are usually aware of these terms.